geode-docs/developing/query_select/the_where_clause.html.md.erb

```diff
@@ -235,6 +235,35 @@ If more than one method can be invoked, the query processor chooses the method w
 
 The query processor uses the runtime types of the parameters and the receiver to determine the proper method to invoke. Because runtime types are used, an argument with a null value has no typing information, and so can be matched with any object type parameter. When a null argument is used, if the query processor cannot determine the proper method to invoke based on the non-null arguments, it throws an `AmbiguousNameException`.
 
+**Methods calls with the `SecurityManager` enabled**
+
+When the `SecurityManager` is enabled,
+<%=vars.product_name%> throws a `NotAuthorizedException` when any
+method not on this list of whitelisted methods is invoked:
+
+- `keySet`, `entrySet`, `values`, `containsKey` or `get` on a `Map`, `Collection`, or `Region` object
```

Comment:
Might it be clearer to begin each of these list items with the object type they refer to? That is, would a reader want to begin with the object type and then work down to the methods? E.g.:
- **On a `Region.Entry` object:** `getKey` or `getValue`
- **On a `Date` or `Timestamp` object:** `after`, `before`, `getNanos`, or `getTime`